platform/commonUI/browse/res/templates/browse/object-properties.html

```diff
@@ -0,0 +1,60 @@
+<!--
+ Open MCT Web, Copyright (c) 2014-2015, United States Government
+ as represented by the Administrator of the National Aeronautics and Space
+ Administration. All rights reserved.
+
+ Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ License for the specific language governing permissions and limitations
+ under the License.
+
+ Open MCT Web includes source code licensed under additional open source
+ licenses. See the Open Source Licenses file (LICENSES.md) included with
+ this source code distribution or the Licensing information page available
+ at runtime from the About dialog for additional information.
+-->
+<div ng-controller="ObjectInspectorController as controller">
+    <ul class="flex-elem grows l-inspector-part">
+        <li>
+            <em class="t-inspector-part-header">Properties</em>
+            <div class="inspector-properties"
+                 ng-repeat="data in metadata"
+                 ng-class="{ first:$index === 0 }">
+                <div class="label">{{ data.name }}</div>
+                <div class="value">{{ data.value }}</div>
+            </div>
+        </li>
+        <li ng-if="contextutalParents.length > 0">
+            <em class="t-inspector-part-header" title="The location of this linked object.">Location</em>
+            <span class="inspector-location"
+                  ng-repeat="parent in contextutalParents"
+                  ng-class="{ last:($index + 1) === contextualParents.length }">
+                <mct-representation key="'label'"
+                                    mct-object="parent"
+                                    ng-model="ngModel"
+                                    ng-click="ngModel.selectedObject = parent"
+                                    class="location-item">
+                </mct-representation>
+            </span>
+        </li>
+        <li ng-if="primaryParents.length > 0">
+            <em class="t-inspector-part-header" title="The location of the original object that this was linked from.">Original Location</em>
+            <span class="inspector-location"
+                  ng-repeat="parent in primaryParents"
+                  ng-class="{ last:($index + 1) === primaryParents.length }">
```

Comment:
@charlesh88 @akhenry Any reason why we use a class for "last" instead of the `last-child` selector? Removing this would reduce watches.

If the `last` class is required, we should at the least use the `$last` boolean provided by ng-repeat.